src/wrappers/lambda.js

```diff
@@ -106,6 +116,9 @@ function baseLambdaWrapper(
             // eslint-disable-next-line no-underscore-dangle
             process._events.beforeExit = originalBeforeExit;
 
+            // Mark trace as sent.
+            tracesSent = true;
```

Comment:
whats the difference between this and callbackCalled?

Reply:
I think I understand the difference now, but it is confusing. Maybe unite `callbackCalled` and `traceSent` to `traceSent` or somea better name or something?